Automattic/wp-calypso
# WordAds: Remove ads settings legacy flux store

#### Changes proposed in this Pull Request

This PR removes the legacy ads settings flux store, as it's no longer necessary once #39802 is merged.

#### Testing instructions

* Checkout this branch.
* Verify the files we're removing here aren't used anywhere.

Part of #18222. Depends on #39802.
